about/press_kit/index.markdown

```diff
@@ -21,54 +23,91 @@ Specific version references of MoveIt framework:
+
+
+## Trademark Style Guidelines ##
 
-## Download
 
-Note for any print media it is necessary to use the provided .AI file and print in the provided CYMK values.
+#### MoveIt in text ####
 
-- Digital Color 0A58F7
-- Print Color (CMYK) C96 M64 Y0 K3
 
-### MoveIt
+*   MoveIt™ should have a trademark symbol the first time it appears in a creative.
+*   MoveIt should always be presented in camel case and is never plural.
+*   Specific versions of MoveIt should be specified by placing the release name or version number after the MoveIt name, separated by a space. For example, MoveIt Melodic, MoveIt Dashing, MoveIt 1, MoveIt 2.
+*   The MoveIt trademarks should not be used in names of companies, organizations, applications, products, or services without the prior written approval of PickNik Robotics. Purely nominative uses (for example: "Our company consults on use of MoveIt." for consultation-based services and "Our company provides training on the use of MoveIt" for training-based services) are acceptable, presuming that the rest of this Policy is followed and there is no express or implied message that PickNik Robotics or the MoveIt project sponsors, endorses, or is otherwise affiliated with your company or its goods and services.
+    *   Incorrect: "MoveIt Deliverbot"
+    *   Correct: "Deliverbot, powered by MoveIt"
+    *   Incorrect: "MoveIt Navomatic"
+    *   Correct: "Navomatic, built on MoveIt"
+    *   Incorrect: "MoveIt-T: A MoveIt Training Initiative by Foo Corp"
+    *   Incorrect: "MoveIt Training Services, Inc."
+    *   Correct: "MoveIt Training Services, offered by Foo Corp"
 
-Dark Backgrounds
-- [PNG](/assets/logo/moveit_logo-white.png)
-- [SVG](/assets/logo/moveit_logo-white.svg)
 
-Light Backgrounds
-- [PNG](/assets/logo/moveit_logo-black.png)
-- [SVG](/assets/logo/moveit_logo-black.svg)
+## MoveIt logo ##
 
-### MoveIt 1
 
-Dark Backgrounds
-- [PNG White](/assets/logo/moveit1/moveit_logo-white.png)
-- [SVG](/assets/logo/moveit1/moveit_logo-white.svg)
+*   The MoveIt logo should have a trademark symbol the first time it appears in a creative.
```

Comment:
Is this a media term like "copy" or did this sentence stop in the middle?

Reply:
If I say moveit(tm) in a YouTube video will I have to say "tm"?  How do I pronounce the upper camel case?

Reply:
I'm curious how the word "creative" is defined.  Could we add a definitions section or some examples?

Reply:
Creative is a legal term (usually used as 'creative work' or 'creative piece' which is why it's left open. I added a link to the Wikipedia definition.